portfolio/src/main/webapp/comment.js

```diff
@@ -12,27 +12,35 @@ function getLoginStatus() {
         let htmlText = json.htmlText;
 
         if (!isLoggedIn) {
```

Comment:
Since they're symmetrical here, maybe convert the negative condition to positive? Negative condition is tiny tini harder to interpret than positive?
```
if (isLoggedin) { 
  do stuff 
} else { 
  do other stuff 
}
```

Unless you only have 1 condition e.g
```
if (!isLoggedIn) {
   // some thing wrong, abort
   return;
}

do something valid
```

Reply:
Fixed